pyega3/pyega3.py

```diff
@@ -15,7 +15,7 @@
 from pyega3.libs.data_file import DataFile
 from pyega3.libs.commands import execute_subcommand
 
-version = "3.4.2"
+version = "3.5.0"
```

Comment:
According to the guidelines at https://semver.org/, I should increase the version to 4.0.0, because I made a backwards incompatible change to the API by removing the `--saveto` parameter:

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

@afoix, @malloryfreeberg, @gariem: what should be the new version?
We can choose from 3.4.3, 3.5.0 and 4.0.0.

Reply:
With my "product owner" hat on, I would vote for 4.0.0 since this is a backwards compatibility-breaking change, and I would prefer to follow the semantic version standards (especially since this is a user-facing tool). Happy to discuss the other options if anyone feels strongly.